### PR TITLE
Add 0.162 release notes

### PR DESCRIPTION
**Pull request by dain**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.162.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+* Fix correctness issue when the value on the left-hand side of an ``IN``
+  expression or a quantified comparison is ``NULL``.
+* Fix correctness issue when the subquery of a quantified comparison produces
+  no rows.
+* Fix correctness issue due to improper inlining of TRY arguments.
+* Fix correctness issue when the right side of a JOIN produces a very large
+  number of rows.
+* Fix correctness issue for expressions with multiple nested boolean
+  conjunctions and disjunctions.
+* Fix performance regression in constructing Hive table layouts.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.162.rst]: This should say "when querying Hive tables with large numbers of partitions". Also, it should probably go in the Hive connector section.

File: presto-docs/src/main/sphinx/release/release-0.162.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.162
+=============
+
+General Changes
+---------------
+* Fix correctness issue when the type of the value in the ``IN`` predicate does
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.162.rst]: Add newline after section headers

File: presto-docs/src/main/sphinx/release/release-0.162.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  not match the type of the elements in the subquery.
+* Fix correctness issue when the value on the left-hand side of an ``IN``
+  expression or a quantified comparison is ``NULL``.
+* Fix correctness issue when the subquery of a quantified comparison produces
+  no rows.
+* Fix correctness issue due to improper inlining of TRY arguments.
+* Fix correctness issue when the right side of a JOIN produces a very large
+  number of rows.
+* Fix correctness issue for expressions with multiple nested boolean
+  conjunctions and disjunctions.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.162.rst]: This is a bit technical. Maybe say "ANDs and ORs" and give an example.

File: presto-docs/src/main/sphinx/release/release-0.162.rst [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  clauses.
+* Improve performance of certain multi-way JOINs by automatically choosing the
+  best evaluation order. This feature is turned off by default and can be
+  enabled via the ``reorder_joins`` config option or session property.
+* Add :func:`xxhash64` and :func:`to_big_endian_64` functions.
+* Add aggregated operator statistics to final query statistics.
+
+SPI Changes
+------------------
+* Connectors can now return optional output metadata for write operations
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.162.rst]: Blank line after section header

File: presto-docs/src/main/sphinx/release/release-0.162.rst [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+* Add ability for event listeners to get connector-specific output metadata.
+* Add client-supplied payload field ``X-Presto-Client-Info`` to ``EventListener``
+
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.162.rst]: Remove extra trailing lines

File: presto-docs/src/main/sphinx/release/release-0.162.rst (hunk 4 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.162.rst]: End item in period

File: presto-docs/src/main/sphinx/release/release-0.162.rst [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  best evaluation order. This feature is turned off by default and can be
+  enabled via the ``reorder_joins`` config option or session property.
+* Add :func:`xxhash64` and :func:`to_big_endian_64` functions.
+* Add aggregated operator statistics to final query statistics.
+
+SPI Changes
+------------------
+* Connectors can now return optional output metadata for write operations
+* Add ability for event listeners to get connector-specific output metadata.
+* Add client-supplied payload field ``X-Presto-Client-Info`` to ``EventListener``
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.162.rst]: End in period

File: presto-docs/src/main/sphinx/release/release-0.162.rst [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+=============
+
+General Changes
+---------------
+* Fix correctness issue when the type of the value in the ``IN`` predicate does
+  not match the type of the elements in the subquery.
+* Fix correctness issue when the value on the left-hand side of an ``IN``
+  expression or a quantified comparison is ``NULL``.
+* Fix correctness issue when the subquery of a quantified comparison produces
+  no rows.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.162.rst]: For these really short ones, I wouldn't wrap, as it makes it harder to read.

File: presto-docs/src/main/sphinx/release/release-0.162.rst [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+* Fix correctness issue due to improper inlining of TRY arguments.
+* Fix correctness issue when the right side of a JOIN produces a very large
+  number of rows.
+* Fix correctness issue for expressions with multiple nested boolean
+  conjunctions and disjunctions.
+* Fix performance regression in constructing Hive table layouts.
+* Improve performance of window functions with similar ``PARTITION BY``
+  clauses.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.162.rst]: Don't wrap here

File: presto-docs/src/main/sphinx/release/release-0.162.rst [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+* Improve performance of certain multi-way JOINs by automatically choosing the
+  best evaluation order. This feature is turned off by default and can be
+  enabled via the ``reorder_joins`` config option or session property.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.162.rst]: The config property is ``reorder-joins`` and the session property is ``reorder_joins``

**Comment by electrum** [comment]
New general change:
```
* Allow specifying column comments for :doc:`/sql/create-table`.

File: presto-docs/src/main/sphinx/release/release-0.162.rst [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+* Fix correctness issue when the type of the value in the ``IN`` predicate does
+  not match the type of the elements in the subquery.
+* Fix correctness issue when the value on the left-hand side of an ``IN``
+  expression or a quantified comparison is ``NULL``.
+* Fix correctness issue when the subquery of a quantified comparison produces no rows.
+* Fix correctness issue due to improper inlining of TRY arguments.
+* Fix correctness issue when the right side of a JOIN produces a very large number of rows.
+* Fix correctness issue for expressions with multiple nested ``AND`` and ``OR`` conditions.
+* Fix performance regression in constructing Hive table layouts.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.162.rst]: Remove this line

**Pull request closed without merging**